sdk/storage/storage-queue/test/utils/tracingutils.ts

```diff
@@ -0,0 +1,359 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { TracerProvider, trace } from "@opentelemetry/api";
```

Comment:
These are centralized in @azure/test-utils on main, but that package does not exist on this branch unfortunately